pkg/controller/logstorage/logstorage_controller.go

```diff
@@ -202,12 +202,11 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return fmt.Errorf("log-storage-controller failed to watch primary resource: %w", err)
 	}
 
-	if err = utils.AddConfigMapWatch(c, render.OIDCUsersConfigMapName, render.ElasticsearchNamespace); err != nil {
-		return fmt.Errorf("log-storage-controller failed to watch the ConfigMap resource: %w", err)
-	}
-
-	if err = utils.AddSecretsWatch(c, render.OIDCUsersEsSecreteName, render.ElasticsearchNamespace); err != nil {
-		return fmt.Errorf("log-storage-controller failed to watch the Secret resource: %w", err)
+	for _, name := range []string{render.OIDCUsersConfigMapName, render.OIDCUsersEsSecreteName,
```

Comment:
Seems like there is a typo in `render.OIDCUsersEsSecreteName`.

Reply:
Yes there is, I'm going to leave it for this PR since it was already there, if that's OK with you.